<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hybrid Protocol GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupCA">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>330</width>
      <height>220</height>
     </rect>
    </property>
    <property name="title">
     <string>Certification Authority (CA)</string>
    </property>
    <widget class="QPushButton" name="btnGenCA">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Generate CA keys</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnIssueCerts">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Issue certs (Alice,Bob)</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainCAInfo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>310</width>
       <height>130</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupAlice">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>520</width>
      <height>380</height>
     </rect>
    </property>
    <property name="title">
     <string>Alice (sender)</string>
    </property>
    <widget class="QLabel" name="labelAliceFiles">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>100</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Alice files</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listAliceFiles">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>240</width>
       <height>120</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelMessage">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>200</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Message to send</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainAliceMessage">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>40</y>
       <width>240</width>
       <height>120</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnReqBobCert">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Request Bob Cert</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnEncrypt">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>170</y>
       <width>110</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Encrypt</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainAliceLog">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>490</width>
       <height>150</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSend">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>170</y>
       <width>110</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBob">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>520</width>
      <height>620</height>
     </rect>
    </property>
    <property name="title">
     <string>Bob (receiver)</string>
    </property>
    <widget class="QLabel" name="labelBobFiles">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>100</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Bob files</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listBobFiles">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>200</width>
       <height>120</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDecrypt">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>40</y>
       <width>110</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Decrypt message</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelDecrypted">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>120</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Decrypted message</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainDecrypted">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>500</width>
       <height>160</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnVerify">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Verify signature</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainBobLog">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>500</width>
       <height>200</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
